hw/top_earlgrey/top_earlgrey_asic.core

```diff
@@ -25,6 +29,8 @@ targets:
   lint:
     <<: *default_target
     default_tool: verilator
```

Comment:
since you're at it, could you copy over all lint and syn settings from here (note: toplevel name has to be changed in the syn target)?

https://github.com/lowRISC/opentitan/blob/e8f2dfd32322b0c12ca17b121960a1e0de7a5e4f/hw/top_earlgrey/top_earlgrey.core#L106-L129